saleor/channel/utils.py

```diff
@@ -0,0 +1,9 @@
+from .models import Channel
+
+
+def raise_channel_validation():
```

Comment:
When we merge #5818 you should use `get_default_channel_slug_if_available` for getting default channel.